nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -96,7 +97,8 @@ controller_server:
     min_y_velocity_threshold: 0.5
     min_theta_velocity_threshold: 0.001
     progress_checker_plugin: "progress_checker"
-    goal_checker_plugin: "goal_checker"
+    goal_checker_plugins: ["precise_goal_checker", "general_goal_checker"]
+    default_goal_checker: "precise_goal_checker" 
```

Comment:
As far as I can tell, this is not a valid parameter read in anywhere or utilized.

Reply:
There's also a question if this should even be a parameterization. The controller plugins don't have a default when multiple exist; it simply rejects all requests lacking specification if > 1 exists on the server.

Reply:
It is fine to me to follow that approach you say. No problem with that on my side.

The only minor issue I see is that the goal_checker takes maybe a bit more importance that It should have for the general case. (I think)

What abut the cases where the goal_checker selector is not being used?

Reply:
Its no different than the controller / planners before the selectors were created - its hardcoded per request or set via a blackboard variable

Reply:
In fact, that parameter was not being used in nav_controller.cpp.
Now it is removed in the current version.